create_test_files.py

```diff
@@ -0,0 +1,101 @@
+from pic2vec import ImageFeaturizer
```

Comment:
This is a script that I ran to generate the CSVs and arrays used in testing again to stay up to date with the library upgrades. It is not called anywhere in the code.

Reply:
Don't just me for the repetitive code :O